src/main/scala/mesosphere/marathon/core/task/tracker/impl/TaskSerializer.scala

```diff
@@ -138,6 +149,24 @@ object TaskSerializer {
   }
 }
 
+object MarathonTaskStatusSerializer {
+  def fromProto(proto: Protos.MarathonTask.MarathonTaskStatus): MarathonTaskStatus = {
+    val result = MarathonTaskStatus.all.collectFirst {
+      case status: MarathonTaskStatus if status.toString == "" + proto => status
+    }
+    // TODO ju<>me discuss
+    result.orNull
```

Comment:
Well .. this _should_ occur only during migration where no MarathonTaskStatus is present in Proto. Afterwards MarathonTaskStatus is mandatory in Task.Status and Proto, therefore the Serializer was intended to handle no `Option`. To indicate my  unhappiness with this `orNull` I added the TODO ... maybe you have an hint to enhance this.